sdk/storage/storage-file-share/src/generated/src/operations/share.ts

```diff
@@ -446,7 +446,7 @@ const createPermissionOperationSpec: coreHttp.OperationSpec = {
       headersMapper: Mappers.ShareCreatePermissionHeaders
     }
   },
-  isXML: false,
```

Comment:
This change should be reverted. The autorest plugin has a known issue of generating this incorrect field.

Reply:
@jeremymeng Any ETA for autorest to have a fix? Really easy to forget this everytime regeneration.

Reply:
Fixed.